src/templates/layouts/default.jade

```diff
@@ -30,12 +32,12 @@ html
 				{ name: 'shapes', data: { url: './sprites/shapes.svg' } }
 			]
 
+		//- Application init
+		script(src='#{jqueryCDN}', defer)
+		script(src='scripts/#{buildstamp}index.js', defer)
```

Comment:
Možná by bylo fajn tam dát async... Ale muselo by se to pořešit, aby se nespustil index bez jQuery (ale to by většinou mělo být v cache). Třeba nějak takhle (nevím moc o ničem lepším):
```js
const i = setInterval(() => {
    if(window.jQuery) {
        clearInterval(i)
        executeIt()
    }
}, 50)
```